Week-01/Day-02_Finding-Nemo/finding_nemo_botiboti/src/lib.rs

```diff
@@ -0,0 +1,39 @@
+#[allow(non_snake_case)]
+
+pub fn findNemo(inp: &str) -> String {
```

Comment:
Nice solution, didn't know about `position` function.
Nice use of Maybe.
However, please split the function into two smaller functions, one to return the index, and one to turn it into a string.

Reply:
Try `Option.mapOr`